### ISSUE TITLE
train_unet

### ISSUE THREAD

**Issue by mjdhelo** [facebookresearch/fastMRI#30]
hallo,
Im a master-student and I was trying to use Unet model to train a model for mri Reconstruction. 
I have an h5 dataet that has my Kspace Information. 
I tried to use the model as it is, but there was always the same error raised, that the path mydata.h5//muticoil.val couldnt be found. 
I tried to creat data_loader before I train the model to give a defined dataloader for my trainer. The same Issue came that the path mydata.h5//multicoil.train  couldn't be found. The error is  being raised in the class SliceData() in the file mri_data at the line 40.
Basically, I assumed that the datatransformer has the goal to change the dataset I am using, but it doesnt do the work, so I thought the problem might be in my data, I even used data from  mridata.org but still the same error is being raised. 
can anyone help me please ?
Thank you
Majd

**Comment by mjdhelo** [comment]
Hallo,
I Understand now, I assumed I had to make special path for my data. It worked, I got another problem. Im on it
